src/qt/bitcoin.cpp

```diff
@@ -453,15 +464,15 @@ int GuiMain(int argc, char* argv[])
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
 #endif
 
-    BitcoinApplication app(*node);
+    BitcoinApplication app;
 
     /// 2. Parse command-line options. We do this after qt in order to show an error if there are problems parsing these
     // Command-line options take precedence:
-    node->setupServerArgs();
+    SetupServerArgs(node_context);
```

Comment:
nit: Could add an unused reference to the argsmanager that is supposed to be used by gui code (instead of the global).

Would either be `const ArgsManager& args = *node_context.args` or `= gArgs`?

Feel free to ingore.